client/src/redux/postsRedux.js

```diff
@@ -12,11 +13,23 @@ export const loadPostsRequest = () => {
     dispatch(startRequest());
     try {
       let res = await axios.get(`${API_URL}/posts`);
-      await new Promise((resolve, reject) => setTimeout(resolve, 2000));
+      await new Promise((resolve, reject) => setTimeout(resolve, 1000));
```

Comment:
Nie dawaj sztucznych opóźniaczy w kodzie zwłaszcza po stronie klienta, gdzie pobrane dane musza być jak najszybciej obsłużone, taka sytuacja, że klient dodatkowo czeka na to jeszcze jest nie do przyjęcia, a o takim kodzie bardzo łatwo zapomnieć aby go usunąć. To samo tyczy się innych miejsc, gdzie użyłeś opóźnienia.

Jeżeli chcesz opóźnienie, to  bez zmian w kodzie , masz możliwość symulacji wolnego łącza w DevTools, a jak chcesz w kodzie to po stronie serwera i najlepiej uwarunkowane od trybu pracy

Reply:
ok.